File: framework/src/main/java/wulinpeng/com/framework/base/ui/loadmore/DefaultFooterViewFactory.java
package wulinpeng.com.framework.base.ui.loadmore;

import android.content.Context;

/**
 * @author wulinpeng
 * @datetime: 18/9/22 下午8:35
 * @description:
 */
public class DefaultFooterViewFactory implements IFooterViewFactory {
    @Override
    public IFooterView createFooterView(Context context) {
        return new DefaultFooterView(context);
    }
}

File: framework/src/main/java/wulinpeng/com/framework/base/ui/loadmore/DefaultFooterViewFactory.kt
package wulinpeng.com.framework.base.ui.loadmore

import android.content.Context

/**
 * @author wulinpeng
 * @datetime: 18/9/22 下午8:35
 * @description:
 */
class DefaultFooterViewFactory : IFooterViewFactory {
    override fun createFooterView(context: Context): IFooterView {
        return DefaultFooterView(context)
    }
}
